src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -250,16 +250,16 @@
                                                                                              Position:absolute to eliminate jump in dojo drag. -->
             <div class="plugin-container">
                 <div class="plugin-container-header">
-                    <h6><%- title %></h6>
+                    <h6 class="i18n" data-i18n="<%- title %>"><%- title %></h6>
                     <a class="plugin-off" href="javascript:;">&#10006;</a>
                     <a class="plugin-close" href="javascript:;">&#95;</a>
 
                     <% if (hasCustomPrint) { %>
-                        <a class="plugin-print" href="javascript:;" title="Print Contents"><i class="fa fa-print"></i></a>
+                        <a class="plugin-print i18n" data-i18n="[title]Print contents" href="javascript:;" title="Print Contents"><i class="fa fa-print"></i></a>
```

Comment:
This doesn't translate correctly because of an inexact string match: should be "Print Contents", is "Print contents"